plotly_resampler/figure_resampler/figure_resampler_interface.py

```diff
@@ -1310,23 +1241,15 @@ def construct_update_data(
             return dash.no_update
 
         # -------------------- construct callback data --------------------------
-        layout_traces_list: List[dict] = []  # the data
-
-        # 1. Create a new dict with additional layout updates for the front-end
-        extra_layout_updates = {}
-
-        # 1.1. Set autorange to False for each layout item with a specified x-range
-        xy_matches = self._re_matches(re.compile(r"[xy]axis\d*.range\[\d+]"), cl_k)
-        for range_change_axis in xy_matches:
-            axis = range_change_axis.split(".")[0]
-            extra_layout_updates[f"{axis}.autorange"] = None
-        layout_traces_list.append(extra_layout_updates)
+        # 1. Create the layout data for the front-end
+        layout_traces_list: List[dict] = [relayout_data]
```

Comment:
Can we type this better? List dictionaries, but what do those dicts hold?

Reply:
Agree - guess using pydantic might (i) enhance the readability while (ii) allowing for validation

Reply:
I'll add a seperate issue for this!